notebooks/Add GPS Metadata Fields_FS.ipynb

```diff
@@ -0,0 +1,266 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    itemID = "7285d87a155649cf93c7826755503f0a" # itemID of feature layer
</pre><p>Should use snake_case for all variable names to be "pythonic" - make sure you update the markdown above too accordingly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/field-maps-scripts/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>